GPU/GLES/GPU_GLES.cpp

```diff
@@ -600,7 +600,7 @@ void GPU_GLES::CheckGPUFeatures() {
 	if (maxVertexTextureImageUnits >= 3) // At least 3 for hardware tessellation
 		features |= GPU_SUPPORTS_VERTEX_TEXTURE_FETCH;
 
-	if (gl_extensions.ARB_texture_float || gl_extensions.OES_texture_float)
+	if (gl_extensions.ARB_texture_float || gl_extensions.OES_texture_float || gl_extensions.OES_texture_half_float)
```

Comment:
According to https://www.khronos.org/registry/OpenGL/extensions/OES/OES_texture_float.txt, OES_texture_half_float only means half-float is supported, not float.  Presumably no device would support half float and not single float, but there's no reason for this check IMHO.

-[Unknown]